docs/home/howto/LOGGING.md

```diff
@@ -259,7 +259,7 @@ Elasticsearch. For non-production environment you can easily disable it by addin
 
 ---
 
-## How to add multiline support for Filebeat logs
+## Multiline support for Filebeat logs
```

Comment:
If you want to change this header you have change reference here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md